libdevcore/CommonTime.h

```diff
@@ -0,0 +1,34 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file CommonTime.h
+ * @author Lefteris Karapetsas <lefteris@ethdev.com>
+ * @date 2015
+ *
+ * Helper functions having to do with time
+ */
+
+#pragma once
+
+#include <time.h>
+
+namespace dev
+{
+
+/// Cross platform wrapper for thread safe gmtime()
+tm *timeToUTC(const time_t *_timeInput, struct tm *_result);
```

Comment:
If it's not used elsewhere this can be added to structured logger rather than another translation unit. Windows #ifdef statements are already popular in Common.h/cpp and future structured logger events will require other definitions from Common.